scripts/west_commands/runners/esp32.py

```diff
@@ -62,6 +62,8 @@ def do_add_parser(cls, parser):
                             help='serial port to flash')
         parser.add_argument('--esp-baud-rate', default='921600',
                             help='serial baud rate, default 921600')
+        parser.add_argument('--esp-monitor-baud', default='115200',
```

Comment:
can you please move this change to a separate PR as it is not related to the C2 support?

Reply:
This is actually for C2. The standard baud rate is 74880. so we need to pass this parameter to `monitor`